### ISSUE TITLE
F.batch_norm uses unbiased estimator but documentation says biased estimator

### ISSUE THREAD

**Issue by elifriedman** [pytorch/pytorch#83256]
### 📚 The doc issue

The documentation for [F.batch_norm](https://pytorch.org/docs/stable/generated/torch.nn.functional.batch_norm.html?highlight=batch_norm#torch.nn.functional.batch_norm) claims that it uses the biased estimator:

> The standard-deviation is calculated via the biased estimator, equivalent to torch.var(input, unbiased=False)

However, a simple test shows that it actually uses the unbiased estimator:

```python
x = torch.rand((100, 3, 10, 10))
unbiased_var = x.var((0, 2, 3), unbiased=True)
biased_var = x.var((0, 2, 3), unbiased=False)
mean = torch.zeros(3)
var = torch.ones(3)
F.batch_norm(x, 
    running_mean=mean, running_var=var, 
    training=True, momentum=1.0)
uses_unbiased_var = torch.allclose(var, unbiased_var)
uses_biased_var = torch.allclose(var, biased_var)
assert uses_biased_var, "Uh oh, batch_norm should use biased var"
```

### Suggest a potential alternative/fix

Update the documentation for [BatchNorm1D](https://pytorch.org/docs/stable/generated/torch.nn.BatchNorm1d.html), [BatchNorm2D](https://pytorch.org/docs/stable/generated/torch.nn.BatchNorm2d.html), and [BatchNorm3D](https://pytorch.org/docs/stable/generated/torch.nn.BatchNorm3d.html) to say that batch_norm uses the unbiased variance estimator:

> The standard-deviation is calculated via the unbiased estimator, equivalent to torch.var(input, unbiased=True)

cc @svekars @holly1238 @albanD @mruberry @jbschlosser @walterddr @kshitij12345 @saketh-are

**Comment by jbschlosser** [comment]
Thanks for the report! Closing as a duplicate of #77427 - we'd still accept a PR addressing this :)
